src/map/GMap.js

```diff
@@ -2,8 +2,8 @@
 (function (root, factory) {
     if (typeof define === "function" && define.amd) {
         var protocol = window.location.protocol === "https:" ? "https:" : "http:";  //  Could be "file:"
-        var __hpcc_gmap_apikey = __hpcc_gmap_apikey || "AIzaSyDwGn2i1i_pMZvnqYJN1BksD_tjYaCOWKg";
-        define(["d3", "../common/HTMLWidget", "../layout/AbsoluteSurface", "async!" + protocol + "//maps.google.com/maps/api/js?key=" + __hpcc_gmap_apikey, "css!./GMap"], factory);
+        var hpcc_gmap_apikey = __hpcc_gmap_apikey || "AIzaSyDwGn2i1i_pMZvnqYJN1BksD_tjYaCOWKg";
```

Comment:
How does this expose the var to the parent html page?  would it need to be
```var __hpcc_gmap_apikey = window["__hpcc_gmap_apikey"] || "AIzaSyDwGn2i1i_pMZvnqYJN1BksD_tjYaCOWKg";```

Reply:
The idea is that it's already defined (if its defined at all) in the global scope of the parent html page. This PR just changes the local scope variable name and I don't see how this code change would behave any differently than it already is? Unless maybe it's overwriting the global variable in a race condition in an older browser?